src/Features/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -11871,5 +11871,37 @@ IEnumerator IEnumerable.GetEnumerator()
                 """,
             }.RunAsync();
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70232")]
+        public async Task TestMissingWhenAlreadyContainingImpl()
```

Comment:
Shouldn't this be in https://github.com/dotnet/roslyn/blob/main/src/Features/CSharpTest/ImplementInterface/ImplementImplicitlyTests.cs ?